clients/java/src/main/java/io/camunda/zeebe/client/impl/worker/JobWorkerImpl.java

```diff
@@ -194,11 +194,16 @@ private void onPollSuccess(final JobPoller jobPoller, final int activatedJobs) {
     // first release, then lookup remaining jobs, to allow handleJobFinished() to poll
     releaseJobPoller(jobPoller);
     final int actualRemainingJobs = remainingJobs.addAndGet(activatedJobs);
-    pollInterval = initialPollInterval;
-    if (actualRemainingJobs <= 0) {
-      schedulePoll();
+
+    if (jobStreamer.isOpen()) {
+      backoff(jobPoller, null);
```

Comment:
:x: We want to back off only if the response is empty, i.e. if `activatedJobs == 0`.